### DISCUSSION TITLE
Un nouveau nom pour la convergence du plugin

### DISCUSSION THREAD

**Discussion by Doctor-Who** [geo2france/idg-qgis3-plugin#14] [Ideas]
Quel nom pour ce plugin unifié ?

**Comment by Doctor-Who** [comment]
J'ai tenté d'utiliser ce site https://www.dcode.fr/generateur-acronyme avec :

- Outil de Chargement Rapide de Couche
- Chargement de Couche Géographique
- Explorateur Infrastructure Données Géographique
- Explorateur de Couche Géographique

Sans grand succès pour le moment.

En espérant que vous aurez plus d'inspiration :)

**Comment by tbrunelle9** [comment]
- Services géographiques des IDG
- Catalogue de données géographiques
 
Je ne suis pas plus inspiré...

**Comment by jbdesbas** [comment]
En lien direct avec le nom (pas encore choisi), 
Le terme IDG dans l'explorateur n'est pas hyper éloquent.
![image](https://user-images.githubusercontent.com/6163107/232033409-214c9857-cb5d-445f-bc0a-889d39de375a.png)

Est-ce qu'on pourrait mettre quelque chose de plus explicite et accessibles,  par exemple `IDG - Géoplateformes` (ou autres proposition ?)

**Reply by gryckelynck** [reply]
Pour le nom du plugin je ne sais pas trop, mais pour l'affichage dans l'explorateur je verrai quelque chose de simple et lisible comme "IDG - Catalogue de données"...
Au passage, faut-il penser d'abord en anglais ou en français ? Pour un nom il me semble que ça peut avoir de l'importance car les termes d'une langue à l'autre n'ont pas toujours la même signification.

**Reply by bchartier** [reply]
> Au passage, faut-il penser d'abord en anglais ou en français ? Pour un nom il me semble que ça peut avoir de l'importance car les termes d'une langue à l'autre n'ont pas toujours la même signification.

Je n'ai rien contre un nom bien franchouillard.
Après, je n'ai rien contre quelque chose qui passe bien en français et en anglais.

**Reply by jbdesbas** [reply]
Le plugin gère l'internationalisation, on peut donc proposer plusieurs affichages (en fonction de la locale paramétrée dans QGIS). D’ailleurs si certains sont chauds pour les traductions en flamand, catalan.. :handshake:

**Reply by bchartier** [reply]
> Le plugin gère l'internationalisation, on peut donc proposer plusieurs affichages (en fonction de la locale paramétrée dans QGIS). D’ailleurs si certains sont chauds pour les traductions en flamand, catalan.. 🤝

Cela pourrait en effet intéresser les plateformes frontalières ou transfrontalières (genre DataGrandEst et GeoRhena).

**Comment by gryckelynck** [comment]
Tant que je suis sur ce fil, quelques questions qui pourraient nous aider à y voir plus claire et identifier des mots-clés pour un titre/nom de plugin:
* Qu'est ce que l'on souhaite mettre en avant ?
* Est-ce la notion d'IDG ? De catalogue ("GeoData Catalog") ? D'agrégation/fédération de données (aggregator) - à noter que l'on peut proposer via le plugin des données qui ne sont pas liées/hébergées à notre IDG ? De facilité d'accès ("Easy access") ?

Qu'en pensez-vous ?

**Reply by Doctor-Who** [reply]
EasyFrenchGeoDataCatalog :smile:

**Reply by bchartier** [reply]
> EasyFrenchGeoDataCatalog 😄

:)
L'idée est là. L'idéal serait de ne pas avoir "Catalog" dedans car je trouve que ça porterait une certaine confusion.

**Reply by bchartier** [reply]
> * Qu'est ce que l'on souhaite mettre en avant ?

L'accès facilité et multiplateformes (au sens des IDG et non des OS).

> * Est-ce la notion d'IDG ? De catalogue ("GeoData Catalog") ? D'agrégation/fédération de données (aggregator) - à noter que l'on peut proposer via le plugin des données qui ne sont pas liées/hébergées à notre IDG ? De facilité d'accès ("Easy access") ?

J'aime bien la notion d'agrégation/fédération.
J'aime bien la notion d'IDG mais c'est peut-être un terme repoussoir ?
